index.bs

```diff
@@ -10154,7 +10163,7 @@ of an {{AudioWorkletNode}} and its associated
 
 	2. In the main scope, <code>context.audioWorklet</code> is requested to add a script module.
 
-	2. Since none exists yet, a new {{AudioWorkletGlobalScope}} is created in association with the context'sThis is the global scope in which {{AudioWorkletProcessor}} class definitions will be evaluated. (On subsequent calls, this previously created scope will be used.)
+	2. Since none exists yet, a new {{AudioWorkletGlobalScope}} is created in association with the context's scope. This is the global scope in which {{AudioWorkletProcessor}} class definitions will be evaluated. (On subsequent calls, this previously created scope will be used.)
```

Comment:
@hoch Can you fill in the missing bits here.  I just put "context's scope", but I would like you to say what this should really say.

Reply:
s/'sThis/. This/